### ISSUE TITLE
unable to retrieve home identifier

### ISSUE THREAD

**Issue by squeeb** [eko/tado-exporter#396]
Not sure if Tado have changed the structure of the returned items, but there appears to be an issue parsing the response data: 

```
Nov 17 21:39:32 prometheus tado-exporter[179281]: [2024-11-17T21:39:32Z INFO  tado_exporter::tado::client] retrieving weather details ...
Nov 17 21:39:33 prometheus tado-exporter[179281]: [2024-11-17T21:39:33Z ERROR tado_exporter::tado::client] unable to retrieve home identifier: error decoding response body: missing field `homes` at line 1 column 72
```

Unsure how to inspect the data being returned, or how to make the exporter spit out verbose information, so this is the only error I have available. 

I was running 1.0.0 when this issue started a few weeks ago, but have since updated the exporter to 1.1.0 but still getting this error and no metrics data.

**Comment by gyulaasztalos** [comment]
We are waiting for this fix to be merged: https://github.com/eko/tado-exporter/pull/393

In the meantime I already applied the fix in my docker image: https://hub.docker.com/r/asztalosgyula/tado-exporter

**Comment by squeeb** [comment]
@eko , https://github.com/eko/tado-exporter/pull/393 appears to work - have just compiled it and the above issue has is resolved.

**Comment by eko** [comment]
Thank you @squeeb @gyulaasztalos @stefano-pogliani,

This is now available in v1.2.0 tag.

Closing this issue for now but feel free to reopen.
